src/parse_file.py

```diff
@@ -276,9 +276,9 @@ def parse_incbin(
     return None
 
 
-def parse_file(f: typing.TextIO, arch: ArchAsm, options: Options) -> MIPSFile:
+def parse_file(f: typing.TextIO, arch: ArchAsm, options: Options) -> AsmFile:
     filename = Path(f.name).name
-    mips_file: MIPSFile = MIPSFile(filename)
+    mips_file: AsmFile = AsmFile(filename)
```

Comment:
if you feel like changing it there's a bunch more instances of "mips" in this file, e.g. this local var name, and also the doc comment at the top of asm_instruction.py.

Reply:
After this I checked with `rg -i mips src/ *.py` and the remaining uses look genuinely MIPS-specific.